tests/subsys/bluetooth/fast_pair/storage/account_key_storage/src/test_corrupted_data.c

```diff
@@ -79,24 +88,30 @@ static void self_test(void)
 	cu_account_keys_validate_loaded(ACCOUNT_KEY_MIN_ID, key_cnt);
 }
 
-static void self_test_rollover(void)
+ZTEST(suite_settings_mock_self_test, test_self_rollover)
 {
+	if ((ACCOUNT_KEY_CNT < 2) &&
+	    IS_ENABLED(CONFIG_BT_FAST_PAIR_STORAGE_OWNER_ACCOUNT_KEY)) {
```

Comment:
`if(IS_ENABLED(CONFIG_BT_FAST_PAIR_STORAGE_OWNER_ACCOUNT_KEY)` ?
The test still does not properly cover case where owner AK and multiple AKs are supported in configuration (because of `next_account_key_id` implementation, we would overwrite owner AK here)